Source/AtomicJS/Javascript/JSAPI.cpp

```diff
@@ -69,13 +69,17 @@ void js_class_declare_internal(JSVM* vm, void* uniqueClassID, const char* packag
         duk_put_prop(ctx, -3);
         duk_pop(ctx);
     }
-    else
+    
+    if (!duk_get_global_string(ctx, package))
     {
-        // RefCounted only supported in Atomic package
-        assert(String("Atomic") == package );
+        // Failure leaves undefined on the stack
```

Comment:
It does. I'm using JSBind on some non-Atomic namespaced code, but the class does extend from RefCounted. When this is loaded, the Atomic namespace object exists, but our namespace object does not.